### PR TITLE
Align files

### PR DESCRIPTION
**Pull request by architectbot**
This PR is generated by https://github.com/giantswarm/github

### CODE REVIEW

**Review by marians — CHANGES_REQUESTED** [review]
The code owner is still wrong

**Comment by TheoBrigitte** [comment]
fixed jwt security issue

File: CODEOWNERS [hunk 1]
```diff
@@ -1,4 +1,4 @@
 # Teams:
-* @giantswarm/team-atlas
+* @giantswarm/team-biscuit
```
Comment (marians) [inline CODEOWNERS:2]: Should be atlas

**Review by marians — APPROVED** [review]

File: CODEOWNERS (hunk 1 above)
Comment (TheoBrigitte) [reply CODEOWNERS]: that's fixed

**Pull request merged**